### PR TITLE
fix: fix practice-two's bug

### PR DESCRIPTION
**Pull request by quanghuy1799**
This PR is use for fix practice-two's bug

### CODE REVIEW

File: practice-two/src/index.html [hunk 1]
```diff
@@ -48,7 +48,7 @@
     </div>
 
     <!-- Modal Add notes -->
-    <div class="modal" id="addnotesmodal" tabindex="-1" role="dialog" aria-labelledby="addnotesmodalTitle"
+    <form class="modal" id="addnotesmodal" tabindex="-1" role="dialog" aria-labelledby="addnotesmodalTitle"
```
Comment (bannguyen-agilityio) [inline practice-two/src/index.html:51]: Should separate html structures and styles between forms and modals

File: practice-two/src/script/modules/addNote.js [hunk 2]
```diff
@@ -11,22 +13,20 @@ const discardButton = document.querySelector('.modal-footer .btn-danger');
...
 
 async function addNoteAndRender() {
-  const title = noteTitleInput.value.trim();
-  const description = noteDescriptionInput.value.trim();
+  const formData = new FormData(noteForm);
+  const title = formData.get('title').trim();
+  const description = formData.get('description').trim();
 
-  if (title && description && selectedCategory) {
+  if (isFormValid(formData)) {
```
Comment (bannguyen-agilityio) [inline practice-two/src/script/modules/addNote.js:29]: Don't you apply from validation schema in this practice?

File: practice-two/src/index.html (hunk 1 above)
Comment (quanghuy1799) [reply practice-two/src/index.html:51]: Yesss Sir!!!

File: practice-two/src/script/modules/addNote.js (hunk 2 above)
Comment (quanghuy1799) [reply practice-two/src/script/modules/addNote.js:29]: I will apply it to the next practice

**Pull request merged**